redirects/juiceshop.md

```diff
@@ -1,7 +1,13 @@
 ---
 
 title: Juice Shop Redirect
-permalink: /juiceshop
+permalink: /juice-shop
+redirect_from:
+  - /juice-shop
+  - /juiceshop
+  - /juice-box
+  - /juicebox
+  - /js
```

Comment:
Do you think `/js`may become problematic in the future if the org has a JavaScript project or initiative? (Or devs/designers need to actually put JS there 😉 )

Reply:
Yeah, that might be an issue, I removed it.